tests/apiclient_test.cc

```diff
@@ -399,7 +399,7 @@ TEST_F(ApiClientTest, InstallDownloadInSeparateInstances) {
     auto result = client.CheckIn();
     ASSERT_EQ(CheckInResult::Status::Ok, result.status);
 
-    auto latest = result.GetLatest();
+    auto latest = result.SelectTarget();
```

Comment:
If we replace all `GetLatest` calls with `SelectTarget` one then there is no test that verifies `GetLatest`. We need either remove `GetLatest` or have some of test invoking it.

Reply:
`GetLatest` is now tested again, as I reverted most of the `GetLatest` -> `SelectTarget` changes.